active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+
+### Inline messages offsetting content
+For InfoBars that are inline with other UI content, keep in mind how the rest of the page will responsively react to the addition of the element.
+
+Messages with a substantial height could dramatically alter the layout of the other elements on the page. 
+If the InfoBar appears or disappears rapidly, especially in succession, the user may be confused with the changing visual state.
+
+# Globalization and Localization
+
+## Color and Icon
+When customizing the color and icon outside of the preset Severity levels, keep in mind user expectations for the connotations from the set of standard icons and colors.
+
+Additionally, the preset Severity colors have already been designed for theme changes, high-contrast mode, color confusion accessibility, and contrast with foreground colors. We recommend to use these colors when possible and to include custom logic in your application to adapt to the various color states and accessibility features.
+
+Please view the UX guidance for [Standard Icons](https://docs.microsoft.com/en-us/windows/win32/uxguide/vis-std-icons) and [Color](https://docs.microsoft.com/en-us/windows/win32/uxguide/vis-color) to ensure your message is communicated clearly and accessible to users.
+
+### Severity
+ Avoid setting the Severity property for a notification that does not match the information communicated in the Title, Message, or custom content.
+ 
+ The accompanying information should aim to communicate the following to use that Severity.
+ - Error: An error or problem that has occurred.
+ - Warning: A condition that might cause a problem in the future.
+ - Success: A long-running and/or background task has completed.
+ - Default: General information that requires the user's attention.
+
+Icons and color should not be the only UI components signifying meaning for your notification. Text in the notification's Title and/or Message should be included to display information.
+
+## Message 
+
+Text in your notification will not be a constant length in all languages. For the Title and Message property this may impact whether your notification will expand to a second line. 
+We recommend you avoid positioning based on message length or other UI elements set to a specific language.
+
+The notification will follow standard mirroring behavior when localized to/from languages that are right to left (RTL) or left to right (LTR). The icon will only mirror if there is directionality.
+
+Please view the guidance for [Adjust layout and fonts, and support RTL](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/adjust-layout-and-fonts--and-support-rtl) for more information about text localization in your notification.
+
+# Remarks
```

Comment:
How are these remarks different than the earlier remarks? Where are there two versions?